MetabolicCompass/View Controller/AuthorizationViewControllers/RegisterViewController.swift

```diff
@@ -104,29 +104,30 @@ private let inputFontSize = ScreenManager.sharedInstance.profileInputFontSize()
     func doConsent() {
         stashedUserId = UserManager.sharedManager.getUserId()
         UserManager.sharedManager.resetFull()
-        ConsentManager.sharedManager.checkConsentWithBaseViewController(self.navigationController!) { [weak self] consentAndNames -> Void in
-            guard consentAndNames.0 else {
+        ConsentManager.sharedManager.checkConsentWithBaseViewController(self.navigationController!) { (consent, firstName, lastName) in
```

Comment:
`navigationController` force unwrapped
`[weak self] `missing by mistake?
`return ()` should gone if block typed correctly